### PR TITLE
docs: reinstate uxdot-example

### LINKED ISSUE #1847: [docs][bug] incorrect image sizes
### Description

Some images contained in `<uxdot-example>` are showing up at 2x the size in the `width-adjustment` attribute.

### Screenshots

Screenshot 2024-09-09 at 3 23 42 PM

### Steps to reproduce

See examples on the [Tooltip - Style subpage](https://ux.redhat.com/elements/tooltip/style/), [Accordion - Overview](https://ux.redhat.com/elements/accordion/#overview), and [Avatar - Overview](https://ux.redhat.com/elements/accordion/#overview)

### Expected behaviour

The image sizes should match what's in the `width-adjustment` attribute.

### Operating System (OS)

macOS

### Browser

_No response_

### Additional browser or assistive technology info

_No response_

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Reinstate `uxdot-example`.

Closes #1847

## Testing Instructions

1.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,7 +9,6 @@ _site
 docs/pfe.min*
 docs/assets/playgrounds/
 docs/assets/javascript/elements/*.js
-!docs/assets/javascript/elements/uxdot-example.js
```
Comment (bennypowers) [inline .gitignore:12]: this is the build artifact, and should be ignored

File: .gitignore [hunk 2]
```diff
@@ -9,7 +9,6 @@ _site
 docs/pfe.min*
 docs/assets/playgrounds/
 docs/assets/javascript/elements/*.js
```
Comment (bennypowers) [inline .gitignore:11]:
```suggestion
docs/assets/javascript/elements/*.js
!docs/assets/javascript/elements/uxdot-example.js
```

File: docs/assets/javascript/elements/uxdot-example.css
Comment (bennypowers) [inline docs/assets/javascript/elements/uxdot-example.css:1]: :see_no_evil: oopsie!

File: .gitignore (hunk 1 above)
Comment (zeroedin) [reply .gitignore:12]: line 11 should be enough to ignore it.  the ! would tell it to not be ignored.

**Review by bennypowers — APPROVED** [review]

**Pull request merged**